main

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
+require './app'
+@app = App.new
+
+def title
+  puts 'Welcome to Microverse School Library!!!'
+  puts
+end
+
+def create_student
+  print 'Enter your name: '
+  name = gets.chomp
+  print 'Enter your age: '
+  age = gets.chomp
+  print 'Enter your classroom: '
+  label = gets.chomp
+  print 'Do you have permission from your parents? [Y/N]: '
+  permission = gets.chomp.downcase
+  if permission == 'y'
+    parent_permission = true
+  elsif permission == 'n'
+    parent_permission = false
+  else
+    puts 'Invalid input'
+  end
+  @app.create_student(age, label, parent_permission, name)
+end
+
+def create_teacher
+  print 'Enter your name: '
+  name = gets.chomp
+  print 'Enter your age: '
+  age = gets.chomp
```

Comment:
You have done a good job adding the UI on the console, however, the file is not executable because the `.rb` extension was omitted in the name. Kindly add the extension in order to make the file executable.

Reply:
Hi @DuaneDave , sorry I was testing the file and forget to get back its extension...Thanks for the review